packages/loot-core/src/server/budget/goaltemplates.js

```diff
@@ -359,6 +353,6 @@ async function applyCategoryTemplate(category, template_lines, month, force) {
       integerToAmount(last_month_balance + to_budget);
     str += ' ' + template_lines.map(x => x.line).join('\n');
     console.log(str);
-    return to_budget;
+    return [to_budget, errors];
```

Comment:
```suggestion
    return { amount: to_budget };
```

Also the code above should be updated to return `{ amount: null }` instead of `null`.

Reply:
What error? (You’ll have to update the code that receives the return value too)